abb_driver/rapid/ROS_stateServer.mod

```diff
@@ -77,13 +77,14 @@ ERROR
     RAISE;  ! raise errors to calling code
 ENDPROC
 
-! signalRobotEStop : System Output
-! signalMotorOn : System Output
+! signalExecutionError : System Output
+! signalExecutionError : System Output

```

Comment:
@gavanderhoorn I know that this is just a commented line, but why do we have two of this signalExecutionError?

Reply:
Because I messed up the rebase. That is all. I'll fix it.

Reply:
Good catch. Should be fixed.